pkg/glusterfs/deploy/kubernetes/csi-provisioner-glusterfsplugin.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 The Gluster CSI Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
```

Comment:
missing space after `#` (maybe also continue leading `#` for the whole comment block....)

Reply:
Yep :)